pillar/edx/ansible_vars/residential.sls

```diff
@@ -81,48 +86,15 @@ edx:
     EDXAPP_YOUTUBE_API_KEY: __vault__::secret-residential/global/edxapp-youtube-api-key>data>value
     EDXAPP_SUPPORT_SITE_LINK: 'https://odl.zendesk.com/hc/en-us/requests/new'
     EDXAPP_CACHE_OPTIONS:
-      'server_max_value_length': {{ memcached_server_max_value_length }}
+      'server_max_value_lengthn': {{ memcached_server_max_value_length }}
```

Comment:
Guessing the `n` at the end of length isn't a typo, but wanted to check

Reply:
Nope, that's a typo...